definitions/private-law/json/AuthorisationCaseField/ManageOrders/AdditionalDocuments/AuthorisationCaseField.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "LiveFrom": "01/08/2023",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "additionalOrderDocuments",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "citizen",
+          "caseworker-privatelaw-solicitor",
+          "caseworker-privatelaw-judge",
+          "caseworker-privatelaw-la",
+          "caseworker-privatelaw-courtadmin",
+          "caseworker-wa-task-configuration",
```

Comment:
wa-task-configuration and citizen can have R access?

Reply:
@shashisk97 replicated existing orderCollection permissions for this.
We can change to R if needed, pls confirm.